specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-08-02-preview/managedClusters.json

```diff
@@ -6979,6 +7021,72 @@
       "required": [
         "enabled"
       ]
+    },
+    "GuardrailsProfile": {
+      "type": "object",
+      "properties": {
+        "systemExcludedNamespaces": {
+          "readOnly": true,
+          "type": "array",
+          "description": "List of namespaces specified by AKS to be excluded from Guardrails",
+          "items": {
+            "type": "string"
+          }
+        },
+        "version": {
+          "type": "string",
+          "description": "The version of constraints to use"
+        },
+        "level": {
+          "type": "string",
+          "enum": [
+            "Off",
+            "Warning",
+            "Enforcement"
+          ],
+          "x-ms-enum": {
+            "name": "level",
+            "modelAsString": true
+          },
+          "description": "The guardrails level to be used"
+        },
+        "includedNamespaces": {
+          "description": "Customer-specified list of namespaces that must be included in guardrails",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "excludedNamespaces": {
+          "description": "Customer-specified list of namespaces that must be excluded from guardrails",
```

Comment:
Similar to above, consider phrasing this like:
- "List of namespaces excluded from guardrails" or 
- "List of namespaces excluded from guardrails checks" or

Reply:
Updated